src/draw/projection.typ

```diff
@@ -0,0 +1,115 @@
+#import "grouping.typ": group, get-ctx
+#import "transformations.typ": set-transform
+#import "/src/process.typ"
+#import "/src/matrix.typ"
+#import "/src/drawable.typ"
+#import "/src/util.typ"
+
+#let ortho-projection-matrix = ((1, 0, 0, 0),
+                                (0, 1, 0, 0),
+                                (0, 0, 0, 0),
+                                (0, 0, 0, 1))
+
+#let ortho-matrix(x, y, z) = matrix.mul-mat(
+  matrix.mul-mat(
+    matrix.mul-mat(
+      matrix.ident(),
+      matrix.transform-rotate-x(x)
+    ),
+    matrix.transform-rotate-y(y)
+  ),
+  matrix.transform-rotate-z(z)
+)
```

Comment:
`matrix.mul-mat` can take separate matrices as positional arguments and mulitply them together like this.